rebasehelper/tests/test_specfile.py

```diff
@@ -199,6 +200,97 @@ def test_find_archive_target_in_prep(self, spec_object):
         target = spec_object.find_archive_target_in_prep('misc.zip')
         assert target == 'test-1.0.2/misc'
 
+    @pytest.mark.parametrize('spec_attributes, kwargs, expected_content', [
+        (
+            {
+                'save': lambda: None,
+                'removed_patches': [],
+                'spec_content': dedent("""
+                    Patch0:    0.patch
+                    Patch1:    1.patch
+
+                    %patchlist
+                    2.patch
+                    3.patch
+
+                    %prep
+                    %patch0 -p0
+                    %patch1 -p1
+                    %patch2 -p0
+                    %patch3 -p1
```

Comment:
I see the point in these `-p` options, but wouldn't it be even more obvious if it was like this?
```
%patch0 -p0
%patch1 -p1
%patch2 -p2
%patch3 -p3
```